### PR TITLE
Various sites

### PR DESCRIPTION
**Pull request by wagonza**

### CODE REVIEW

**Review by spirillen — CHANGES_REQUESTED** [review]
I do not as such have anything against you submit both url and domain, I'll guess one of them should be sufficient.

@mitchellkrogza it yours to decide if both are useful.

Note: To see the Phishing site in question, use this url: `https://alignmentlidjsonpanz.web.app/login.html`

<details><summary>Screenshot</summary>

image

</details>

**Comment by wagonza** [comment]
> I do not as such have anything against you submit both url and domain, I'll guess one of them should be sufficient.
> 
> @mitchellkrogza it yours to decide if both are useful.
> 
> Note: To see the Phishing site in question, use this url: `https://alignmentlidjsonpanz.web.app/login.html`

Removed the link and will submit one in future.

File: add-link [hunk 1]
```diff
@@ -40,3 +40,5 @@ http://eternalstarculture.com/intern/home.php
 https://tinyurl.com/4cbf4wcp
 https://rachelleckman.com/.pdf
 https://condocopia.org/contact.html
+https://carlicence.co.za/wp-admin/shiplabel/
```
Comment (spirillen) [inline add-link:43]: Seems fixed to me

image

File: add-domain [hunk 2]
```diff
@@ -521,3 +521,4 @@ mailbox_0upgrade.godaddysites.com
 www-rsblox.com
 pdfkinghub.xyz
 randomzee06.s3.au-syd.cloud-object-storage.appdomain.cloud
+data-base.cf
```
Comment (spirillen) [inline add-domain:524]: I can neither confirm or not. the `https://whois.freenom.com/en/whois.html` seems currently down 

drill data-base.cf @9.9.9.9
;; ->>HEADER<<- opcode: QUERY, rcode: NXDOMAIN, id: 43037
;; flags: qr rd ra ; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 0 
;; QUESTION SECTION:
;; data-base.cf.        IN      A

;; ANSWER SECTION:

;; AUTHORITY SECTION:
cf.     5       IN      SOA     a.ns.cf. kohilaire.socatel.cf. 1625949359 10800 3600 604800 5

**Review by spirillen — CHANGES_REQUESTED** [review]

**Comment by spirillen** [comment]
Can't find any active urls domains

**Pull request closed without merging**